### PR TITLE
throw error when enabled service cannot be loaded

### PR DESCRIPTION
**Pull request by wlan0**
@ibuildthecloud 
Fixes https://github.com/rancherio/os/issues/209

### CODE REVIEW

File: util/util.go [hunk 1]
```diff
@@ -272,7 +272,7 @@ func LoadResource(location string, network bool, urls []string) ([]byte, error)
 		}
 		defer resp.Body.Close()
 		return ioutil.ReadAll(resp.Body)
-	} else if strings.HasPrefix(location, "/") {
+	} else if strings.HasPrefix(location, "/var/lib/rancher/conf") {
```
Comment (ibuildthecloud) [inline util/util.go]: Please don't change this method.  This is a generic method to load files.  I'd rather you check the condition up front in the enable code.

Comment (wlan0) [inline util/util.go]: ok

**Comment by wlan0** [comment]
@ibuildthecloud fixed!

File: cmd/control/service.go [hunk 2]
```diff
@@ -61,6 +63,14 @@ func enable(c *cli.Context) {
 
 	for _, service := range c.Args() {
 		if val, ok := cfg.ServicesInclude[service]; !ok || !val {
+			if !strings.HasPrefix(service, "/var/lib/rancher/conf") {
```
Comment (ibuildthecloud) [inline cmd/control/service.go]: Service "foo" or "http://blah" will fail here.

Comment (wlan0) [inline cmd/control/service.go]: Oops, forgot to push the full changes. Now, its ready

**Comment by wlan0** [comment]
@ibuildthecloud  I changed this PR to stage all containers when service is enabled.

**Comment by ibuildthecloud** [comment]
Can you submit the original simple input validation as a PR.  Then open a separate one to do the pull.  Half of the pull PR needs to go to rancher-compose.

**Comment by wlan0** [comment]
hmmm, ok. But it'll be a while. You just refactored rancher-compose and i have not tested my changes with the refactored rancher-compose

**Comment by ibuildthecloud** [comment]
All of those changes are on master already of rancher-compose and rancheros

**Comment by wlan0** [comment]
@ibuildthecloud Fixed again!

**Comment by ibuildthecloud** [comment]
You're missing the loadresource you had before.

**Comment by ibuildthecloud** [comment]
Also can you make sure the command exits with 1 of it can't turn on the service.

**Comment by wlan0** [comment]
@ibuildthecloud  I fixed it. Why should we return 1?

File: cmd/control/service.go [hunk 3]
```diff
@@ -71,6 +82,7 @@ func enable(c *cli.Context) {
 			log.Fatal(err)
 		}
 	}
+	os.Exit(1)
```
Comment (ibuildthecloud) [inline cmd/control/service.go]: Remove

File: cmd/control/service.go [hunk 4]
```diff
@@ -61,6 +64,14 @@ func enable(c *cli.Context) {
 
 	for _, service := range c.Args() {
 		if val, ok := cfg.ServicesInclude[service]; !ok || !val {
+			if strings.HasPrefix(service, "/") && !strings.HasPrefix(service, "/var/lib/rancher/conf") {
+				log.Printf("ERROR: Service should be in path /var/lib/rancher/conf")
```
Comment (ibuildthecloud) [inline cmd/control/service.go]: log.Fatal

File: cmd/control/service.go [hunk 5]
```diff
@@ -61,6 +64,14 @@ func enable(c *cli.Context) {
...
+				continue
+			}
+			if _, err := docker.LoadServiceResource(service, true, cfg); err != nil {
+				log.Printf("could not load service %s", service)
```
Comment (ibuildthecloud) [inline cmd/control/service.go]: log.Fatal

**Comment by wlan0** [comment]
@ibuildthecloud  Fixed

**Pull request merged**